requirements.txt

```diff
@@ -1,4 +1,4 @@
 isodate
-jsonschema
+jsonschema==3.2.0
```

Comment:
Would maybe `>=3.2.0` be an option or is development so fast that users do not need wait  long until a dependency is bumped? … and maybe sync the build data files also? I noticed that `setup.cfg` still has `jsonschema>=3.0.0`